apps/cowswap-frontend/src/modules/bridge/pure/TokenAmountDisplay/index.tsx

```diff
@@ -25,9 +31,17 @@ export function TokenAmountDisplay({
   usdValue,
   hideFiatAmount = false,
   tokenLogoSize,
+  status,
   libTokenAmountProps,
+  hideTokenIcon = false,
+  parsedAmount: providedParsedAmount,
```

Comment:
We don't need both `amount` and `parsedAmount`.
`amount` must always by `CurrencyAmount` instance.

Reply:
Refactored code now to not use (delete) this hook anymore

Reply:
Refactored this now